content/_data/changelogs/weekly.yml

```diff
@@ -1562,6 +1562,55 @@
     # pull: 3153 too minor: Grammar fixes
     # pull: 3151 internal: Jenkinsfile
     # pull: 3158 internal: disable flaky test
+- version: "2.93"
+  date: 2017-12-03
+  changes:
+    - type: rfe
+      pull: 3135
+      message: >
+        Use Java NIO to read and write Unix file permissions by default.
+        The previous behavior can be restored by setting the Java system property <code>hudson.Util.useNativeChmodAndMode</code> to <code>true</code>.
+      references:
+        - issue: 36088
+        - url: https://wiki.jenkins-ci.org/display/JENKINS/Features+controlled+by+system+properties
+          title: Jenkins features controlled by system properties
+    - type: rfe
+      pull: 3167
+      message: >
+        Better handling of certain unreproducible XML file load/save errors.
+    - type: rfe
+      issue: 47429
+      pull: 3150
+      message: >
+        Improve user lookup performance, for example from Git changelog calculation.
+    - type: rfe
+      issue: 48348
+      pull: 3157
+      message: >
+        Reduce memory usage when scheduling pipelines on big clusters.
+    - type: rfe
+      issue: 34855
+      pull: 2548
+      message: >
```

Comment:
FTR, as explained on the JIRA, this is still open as the fix done was split to still move forward and reduce the risk. The final fix is up as https://github.com/jenkinsci/jenkins/pull/3170 (where, that time, I put the _FIX_ prefix on purpose).